SportsGoods.Data/Repositories/ProductRepository.cs

```diff
@@ -43,6 +43,13 @@ public async Task<PagedResult<Product>> GetPagedAsync(int pageNumber, byte pageS
               .Take(pageSizeInt)
               .ToListAsync();
 
+            var brands = await _context.Brands.ToListAsync();
```

Comment:
this can be slow as well 
if we have too many brands use 

```
var brandIds = products.Select(x => x.BrandId).ToList();
var brands = await _context.Brands.Where(x => brandIds.Contains(x.Id)).ToListAsync();
products.ForEach(product => product.Brand = brands.FirstOrDefault(b => b.Id == product.BrandId));
```

Reply:
@LVarsaminov